sql/python/sqlflow/template/db.py

```diff
@@ -0,0 +1,177 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import abc
+import six
```

Comment:
SQLFlow only supports Python3. I don't think `__future__` is necessary.

Reply:
Ok